apps/term-apply/pkg/s3file/s3file.go

```diff
@@ -0,0 +1,121 @@
+package s3file
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+)
+
+func CopyFromS3(key, filename string) error {
+	bucket, ok := os.LookupEnv("TA_BUCKET")
```

Comment:
Is this reasonable to try to set at main.go? https://github.com/Nebulaworks/orion/blob/5d3155c53e0c262560368b269530d74e858b3be9/apps/term-apply/main.go#L16 Maybe a follow up for later? But just thinking its easier to know all the environment vars if there in the same place

Reply:
also part of inf 515

Reply:
Figured wed be on the same page :laughing: